packages/expo-av/src/Audio.ts

```diff
@@ -28,25 +28,38 @@ const _isValueValid = (value: any, validValues: any[]): boolean => {
 };
 
 // Returns array of missing keys in object. Returns an empty array if no missing keys are found.
-const _findMissingKeys = (object: Object, requiredKeys: any[]): any[] => {
-  return requiredKeys.filter(requiredKey => !(requiredKey in object));
+const _populateMissingKeys = (userAudioMode: Object, defaultAudioMode: AudioMode): AudioMode => {
+  for (let key in defaultAudioMode) {
+    if (!userAudioMode.hasOwnProperty(key)) {
+      userAudioMode[key] = defaultAudioMode[key];
+    }
+  }
+  return userAudioMode as AudioMode;
 };
 
-export async function setAudioModeAsync(mode: AudioMode): Promise<void> {
-  const missingKeys = _findMissingKeys(mode, [
-    'allowsRecordingIOS',
-    'interruptionModeIOS',
-    'playsInSilentModeIOS',
-    'staysActiveInBackground',
-    'interruptionModeAndroid',
-    'shouldDuckAndroid',
-    'playThroughEarpieceAndroid',
-  ]);
-  if (missingKeys.length > 0) {
-    throw new Error(
-      `Audio mode attempted to be set without the required keys: ${JSON.stringify(missingKeys)}`
-    );
+const defaultMode: AudioMode = {
+  allowsRecordingIOS: false,
+  interruptionModeIOS: INTERRUPTION_MODE_IOS_MIX_WITH_OTHERS,
+  playsInSilentModeIOS: false,
+  staysActiveInBackground: false,
+  interruptionModeAndroid: INTERRUPTION_MODE_ANDROID_DUCK_OTHERS,
+  shouldDuckAndroid: true,
+  playThroughEarpieceAndroid: true,
+};
+
+let currentAudioMode: any = null;
```

Comment:
```suggestion
let currentAudioMode: AudioMode | null = null;
```

Reply:
Awesome! Thanks for the help and suggestions on this 😄